src/test/java/com/jesusgsdev/services/CustomerServiceTest.java

```diff
@@ -0,0 +1,61 @@
+package com.jesusgsdev.services;
+
+import com.jesusgsdev.entities.Customer;
+import com.jesusgsdev.repositories.CustomerRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentMatchers;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import java.util.Optional;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.BDDMockito.given;
+
+
+
+public class CustomerServiceTest {
+
+    @InjectMocks private CustomerService customerService;
```

Comment:
Move private CustomerService customerService; to a new line so the annotation are always on top Same for the @Mock below